models/fivetran_log__audit_table.sql

```diff
@@ -1,3 +1,14 @@
+{{ config(
+    materialized='incremental',
+    partition_by={
+        'field': 'sync_start',
+        'data_type': 'timestamp',
+        'granularity': 'day'
```

Comment:
Is day the ideal granularity to partition by? I remember with Zendesk and Jira we encountered some instances where we were over partitioning. Can we confirm that this is the best partition to use?

Reply:
Can we ensure we do not need a merge key in this incremental strategy? 🤔 

I worry that not using a merge key could result in duplicate rows.

Reply:
Maybe we consider insert-override?

Reply:
added a merge key! and looking at log connector data, it looks like it only includes data from after it was set up (which earliest it could be for anyone is [march 2020](https://fivetran.com/docs/logs/fivetran-log/changelog#march2020) when the active_volume table was created. so i think no one would reach the 4k+ daily partition limit